app/prism/Prism.scala

```diff
@@ -72,4 +72,13 @@ object Prism {
       launchConfigurations <- Attempt.sequence(jsons.map(extractLaunchConfiguration))
     } yield launchConfigurations
   }
+
+  def getAccounts(implicit config: AMIableConfig, ec: ExecutionContext): Attempt[List[AWSAccount]] = {
+    val url = s"${config.prismUrl}/sources/accounts"
+    for {
+      response <- Http.response(config.wsClient.url(url).get(), "Unable to fetch accounts list", url)
+      jsons <- accountsResponseJson(response)
+      accounts <- Attempt.sequence(jsons.map(extractAccounts))
```

Comment:
You're copying from the existing code so this is fine as-is. However, I have learned since I first built Amiable that `M.sequence` is rarely useful, with `M.traverse` almost always being the better choice.

In this case:

```scala
accounts <- Attempt.sequence(jsons.map(extractAccounts))
```
Would be better written as:
```scala
accounts <- Attempt.traverse(jsons)(extractAccounts)
```

👍  for consistency though, this is just an FYI for anyone reading this PR.

Reply:
oh that's a lot nicer, I'll keep in mind for future PRs